common/views/components/WorkTags/WorkTags.js

```diff
@@ -0,0 +1,85 @@
+import styled from 'styled-components';
+import { spacing, font, classNames } from '../../../utils/classnames';
+import NextLink from 'next/link';
+import type { NextLinkType } from '@weco/common/model/next-link-type';
+
+export type WorkTagType = {|
+  query: string,
+  textParts: string[],
+  linkAttributes: NextLinkType,
+|};
+
+const WorkTag = styled.div`
+  border-radius: 3px;
+  text-decoration: none;
+  padding: 0.2em 0.5em;
```

Comment:
are we meant to have this on the theme somewhere and using our spacing helpers?

Why the move to `em`? Sure the reason's good, but maybe we could benefit everywhere (chanting the `1 way` mantra ⛩ ).

Reply:
Good question – I always feel that keeping elements that should scale in line with typography (i.e. the amount of space within the tag) in `em`s is a good approach, but appreciate that's not necessarily the path we've gone down with out spacing classes. Unfortunately, `6px` is too much top/bottom padding for these (the design wants them to be more like `4px` once we've set the `line-height` to 1). So do we introduce some extra spacing classes?

Reply:
This makes tonnes of sense, as mentioned could be worth spiking on our current setup and testing on thses and the Buttons?